src/MainScene.tsx

```diff
@@ -32,7 +32,8 @@ export function MainScene(): JSX.Element {
   const isInitialRender = useRef(true) // Create a mutable ref object to keep track of initial render
   const [usdToCoinRates, setUsdToCoinRates] = useState(initRates) // State variable to keep track of the exchange rates
   const [showCodeScreen, setShowCodeScreen] = useState(false) // State variable to determine whether to show the code screen
-  const [coinSelection, setCoinSelection] = useState('') // State variable that keeps track of coin selection
+  const [coinSelection, setCoinSelection] = useState('BTC') // State variable that keeps track of coin selection
```

Comment:
Get the default currency from the `config` file.

Reply:
No longer going to use a default currency. Will revert back to how it was previously with an empty string.